code/modules/detectivework/detective_work.dm

```diff
@@ -75,7 +75,7 @@
 
 /turf/add_blood_DNA(list/blood_dna, list/datum/disease/diseases)
 	var/obj/effect/decal/cleanable/blood/splatter/B = locate() in src
-	if(!B)
+	if(!B || QDELETED(B)) // SKYRAT EDIT - SANITY CHECK; DONT ADD DNA TO A FUCKING DELETED BLOOD SPATTER - original: if(!B)
```

Comment:
```suggestion
	if(QDELETED(B)) // SKYRAT EDIT - SANITY CHECK; DONT ADD DNA TO A FUCKING DELETED BLOOD SPATTER - original: if(!B)
```
QDELETED includes the !B, pardon the typo on the previous~

Reply:
You owe me snuggles